Sources/Commands/UserToolchain.swift

```diff
@@ -112,6 +112,15 @@ public struct UserToolchain: Toolchain {
 
         // Get the binDir from destination.
         let binDir = destination.binDir
+        
+        #if LIB_64
+        let libDir = binDir.parentDirectory.appending(components: "lib64", "swift", "pm")
+        #elseif LIB_32
+        let libDir = binDir.parentDirectory.appending(components: "lib32", "swift", "pm")
+        #else
+        let libDir = binDir.parentDirectory.appending(components: "lib", "swift", "pm")
+        #endif
```

Comment:
I think we can compute the libdir component here and then create the path with that component. Something like:

```swift
#if LIB_64
let libDirComponent = "lib64"
...
#endif
...
libDir: binDir.parentDirectory.appending(components: libDirComponent, "swift", "pm"),
```

Reply:
I will change to me like that.